app/uk/gov/hmrc/gform/gform/SectionRenderingService.scala

```diff
@@ -1171,14 +1171,69 @@ class SectionRenderingService(
         row match {
           case _: MiniSummaryRow.HeaderRow => List(row) :: acc
           case _: MiniSummaryRow.ValueRow  => (row :: acc.head) :: acc.tail
+          case _: MiniSummaryRow.ATLRow    => List(row) :: acc
         }
       )
       .reverse
       .map(_.reverse)
     val htmls = visibleRowsPartitioned.map {
       case MiniSummaryRow.HeaderRow(h) :: xs =>
         HtmlFormat.fill(List(header(Html(sse(h, false))), loop(xs)))
+      case MiniSummaryRow.ATLRow(atlId, _, atlRows) :: xs =>
+        def f1() = {
+          val addToListIteration = ei.formModelOptics.formModelVisibilityOptics.formModel.brackets
+            .addToListBracket(AddToListId(atlId))
+            .iterations
+
+          val summaryListRecords =
+            addToListIteration.toList.map { iteration =>
+              val index = iteration.repeater.repeater.index
+              val fcs: List[FormComponent] = iteration.singletons.toList.flatMap { singletonWithNumber =>
+                val page = singletonWithNumber.singleton.page
+                page.fields.filterNot(_.hideOnSummary)
+              }
+              val updatedRows = atlRows.map {
+                case MiniSummaryRow
+                      .ValueRow(key, MiniSummaryListValue.Reference(FormCtx(formComponentId)), includeIf) =>
+                  val fId =
+                    fcs.find(_.baseComponentId == formComponentId.baseComponentId).map(_.id).getOrElse(formComponentId)
+                  MiniSummaryRow.ValueRow(
+                    key.map(_.expand(index, List())),
+                    MiniSummaryListValue.Reference(FormCtx(fId)),
+                    includeIf.map { case IncludeIf(boolExpr) =>
+                      IncludeIf(boolExpr.replaceExpr {
```

Comment:
not sure if this is the best way to deal with `includeIf` in ATL
I replace all `FormCtx(fcId)` with `FormCtx(fcIndexedId)` if `fcIndexedId.baseId` is available it's baseComponentid is equal to fcId's baseComponentId